### ISSUE TITLE
Opacity is hardcoded

### ISSUE THREAD

**Issue by Lemmmy** [SidebarJS/sidebarjs#7]
Add an option to change the opacity? It's currently hardcoded to 0.3

**Comment by lorenzodianni** [comment]
Hi @Lemmmy, thanks for your feedback.

The intention of this library is to simulate the native sidebar/sidenav of mobile devices.
As you can see on mobile devices, the native sidebar/sidenav has a soft opacity on the backdrop, so i'd like to avoid options that may change the "native" style.

Anyway, if you want a different opacity, you can change it with a simple css override:
```css
[sidebarjs].sidebarjs--is-visible [sidebarjs-background] {
    opacity: 1;
}
```

**Comment by Lemmmy** [comment]
The problem is, it's still hardcoded to 0.3 in the javascript, so while dragging the sidebar open, it will fade from 0 to 0.3, then suddenly jump to 1.

**Comment by lorenzodianni** [comment]
Ok now it's more clear, i'm working on
It will be in the next release 👍

**Comment by lorenzodianni** [comment]
@Lemmmy I've just released v4.0.0 on [npm](https://www.npmjs.com/package/sidebarjs), have fun!

**Comment by Lemmmy** [comment]
Thank you very much!
